tgui/packages/tgui/interfaces/CrewManifest.js

```diff
@@ -0,0 +1,89 @@
+import { classes } from 'common/react';
+import { useBackend } from '../backend';
+import { Icon, Section, Table, Tooltip } from '../components';
+import { Window } from '../layouts';
+
+const commandJobs = [
+  'Head of Personnel',
+  'Head of Security',
+  'Chief Engineer',
+  'Research Director',
+  'Chief Medical Officer',
+];
+
+export const CrewManifest = (props, context) => {
+  const {
+    data: { manifest, positions },
+  } = useBackend(context);
+
+  return (
+    <Window title="Crew Manifest" width={350} height={500}>
+      <Window.Content scrollable>
+        {Object.entries(manifest).map(([dept, crew]) => (
+          <Section
+            className={'CrewManifest--' + dept}
+            key={dept}
+            title={
+              dept
+              + (dept !== 'Misc'
+                ? ` (${positions[dept].open} positions open)`
+                : '')
+            }>
+            <Table>
+              {Object.entries(crew).map(([crewIndex, crewMember]) => (
+                <Table.Row key={crewIndex}>
+                  <Table.Cell className={'CrewManifest__Cell'}>
+                    {crewMember.name}
+                  </Table.Cell>
+                  <Table.Cell
+                    className={classes([
+                      'CrewManifest__Cell',
+                      'CrewManifest__Icons',
+                    ])}
+                    collapsing>
+                    {positions[dept].exceptions.includes(crewMember.rank) && (
+                      <Tooltip content="No position limit" position="bottom">
+                        <Icon className="CrewManifest__Icon" name="infinity" />
+                      </Tooltip>
+                    )}
+                    {crewMember.rank === 'Captain' && (
```

Comment:
I don't like that this is hardcoded for the captain at all - but since we lack JOB_KEY atm I'm gonna allow it